src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,28 @@
+package duke.command;
+
+import duke.Storage;
+
+import duke.task.TaskList;
+
+/**
+ * An abstract class that represents the various types of commands that can be entered by the user.
+ */
+public abstract class Command {
+    private String command;
```

Comment:
Perhaps consider making `command` `protected` instead of `private`? It would make the code much cleaner since subclasses make use of this variable a lot